common/thorhelper/roxiehelper.cpp

```diff
@@ -1190,6 +1201,25 @@ void FlushingJsonBuffer::startScalar(const char *resultName, unsigned sequence)
         tail.set("}]}");
     }
 }
+
+void FlushingJsonBuffer::setScalarInt(const char *resultName, unsigned sequence, __int64 value, unsigned size)
+{
+    startScalar(resultName, sequence);
+    if (size < 7)
```

Comment:
Should have a comment: // JavaSCript only supports 53 significant digits, or whatever the reason is...